tests/integration/secretsmanager/test_secretsmanager.py

```diff
@@ -1709,3 +1713,36 @@ def test_delete_non_existent_secret_returns_as_if_secret_exists(self, sm_client)
         assert response["Name"] == secret_id
         assert response["ARN"] is not None
         assert response["DeletionDate"] is not None
+
+    def test_exp_raised_on_creation_of_secret_scheduled_for_deletion(self, sm_client):
+        create_secret_req: CreateSecretRequest = CreateSecretRequest(
+            Name=f"secret-{short_uid()}", SecretString=f"secretstr-{short_uid()}"
+        )
+        stage_deletion_req: DeleteSecretRequest = DeleteSecretRequest(
+            SecretId=create_secret_req["Name"], RecoveryWindowInDays=7
+        )
+
+        sm_client.create_secret(**create_secret_req)
```

Comment:
nit: as a personal preference, I would directly pass the atomic arguments here instead of creating a `CreateSecretRequest` and later unpacking it.